nodejs/eks/index.ts

```diff
@@ -12,6 +12,8 @@
 // See the License for the specific language governing permissions and
 // limitations under the License.
 
-export { Cluster } from "./cluster";
+export { Cluster, ClusterNodeGroupOptions } from "./cluster";
+export { createNodeGroup, NodeGroup } from "./nodegroup";
+export { createDashboard } from "./dashboard";
```

Comment:
For now at least, I think we can hold off on exposing these functions, and can bring them back publicly using the same pattern as NodeGroup - a component and potentially also an instance function on the cluster.  This is more consistent with our general API style here and in other libraries.  I've limited the exposed surface area a bit here in #49 to give us options for future API design.